tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -444,9 +444,20 @@ def main():
       'and copy two debug builds, one with ASAN and one without.'
   )
 
+  # TODO(http://fxb/110639): Deprecate this in favor of multiple runtime parameters
+  parser.add_argument(
+      '--skip-remove-buckets',
+      action='store_true',
+      default=False,
+      help='This allows for multiple runtimes to exist in the default bucket directory. If '
+      'set, will skip over the removal of existing artifacts in the bucket directory '
+      '(which is the default behavior).'
+  )
+
   args = parser.parse_args()
-  RemoveDirectoryIfExists(_bucket_directory)
   build_mode = args.runtime_mode
+  if (not args.skip_remove_buckets):
```

Comment:
I'm not sure, but this might create issues for uploading the runner to CIPD, since we call build_fuchsia_artifacts.py two times sequentially in the same recipe (engine.py):

1. https://cs.opensource.google/flutter/recipes/+/main:recipes/engine/engine.py;l=225
2. https://cs.opensource.google/flutter/recipes/+/main:recipes/engine/engine.py;l=1041

Have you traced through the logic to verify that these changes should be safe?

If you want to test it, I'm not sure how to test changes to the engine repo alongside the recipes. One option might be to make a copy of build_fuchsia_artifacts.py in the recipes repo and change the engine.py and femu_test.py recipes to use that copy instead.

Reply:
NVM I'm being dumb, this change itself won't affect anything, and it lets us test the behavior in the recipes repo after submission.